data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -331,6 +335,7 @@
     (StringFieldsSuppression,),
     (AggregateZipCodes,),
     (DeidentifyAIANZip3Values,),
+    (DropOrphanedSurveyConductIds,),
     (DropOrphanedPIDS,),
     (RemoveExtraTables,),  # Should be last cleaning rule to be run
```

Comment:
I know the ticket didn't ask you to, but I am going to update it.  I think we should also run this rule as part of the `REGISTERED_TIER_DEID_CLEAN_CLEANING_CLASSES` and `CONTROLLED_TIER_DEID_CLEAN_CLEANING_CLASSES` lists after `DropZeroConceptIDs` and before `CleanMappingExtTables`.